apps/rush/src/start.ts

```diff
@@ -35,6 +35,9 @@ if (process.argv[2] === RUSH_PURGE_OPTION_NAME) {
     const rushWrapper: RushWrapper = versionManager.ensureRushVersionInstalled(configuration.rushVersion);
     rushWrapper.invokeRush();
   } else {
-    Rush.launch(currentPackageJson.version, false);
+    Rush.launch(
+      currentPackageJson.version,
+      !!configuration && configuration.rushVersion === currentPackageJson.version
```

Comment:
>!!configuration && configuration.rushVersion === currentPackageJson.version [](start = 6, length = 75)

Create a local variable called isManaged and explain what you're calculating.  Otherwise this line is pretty cryptice